elixir/content.md

```diff
@@ -34,5 +34,5 @@ iex(snode@elixir)2> :c.uptime
 ## Run a single Elixir exs script
 
 ```console
-$ docker run -it --rm --name %%REPO%%-inst1 -v "$PWD":/usr/src/myapp -w /usr/src/myapp %%REPO%% elixir your-escript.exs
+$ docker run -it --rm --name elixir-inst1 -v "$PWD":/usr/src/myapp -w /usr/src/myapp elixir elixir your-escript.exs
```

Comment:
this line I think may not need to change; because the content on https://hub.docker.com/_/elixir/ is result from somewhat transpile, `%%REPO%%` will be replaced to the actual repo name;

in [comment](https://github.com/c0b/docker-elixir/pull/21#issuecomment-270553222) I mean we can add a new section to describe the new features of tk libraries and `observer`, add some live examples of how `observer` can be used; all this can be pending till after docker-library/official-images#2512 is merged
we don't usually update the docs repo for only version bumps, just don't have to, but when we do, it could be some major features enhancement or like a major version bump from docs being really too far outdate